server/services/transport/daily.mdx

```diff
@@ -916,6 +916,97 @@ See [update_subscriptions](https://reference-python.daily.co/api_reference.html#
 
 </ResponseField>
 
+### Permissions Management
+
+When using the DailyTransport, if you provide a token with owner permissions to the bot, it can control permissions for other participants in the session. With this control, the bot can modify settings for how others send and receive media, how they are seen in a session, and whether they have Administrative controls.
```

Comment:
```suggestion
When using the DailyTransport, if you provide a token with [owner](https://docs.daily.co/reference/rest-api/meeting-tokens/config#is_owner) or [participant admin](https://docs.daily.co/reference/rest-api/meeting-tokens/config#canAdmin) permissions to the bot, it can control permissions for other participants in the session. With this control, the bot can modify settings for how others send and receive media, how they are seen in a session, and whether they have administrative controls.
```

Reply:
Updated. Commit incoming.